packages/component-base/src/tabindex-mixin.js

```diff
@@ -22,23 +22,22 @@ export const TabindexMixin = (superclass) =>
       return {
         /**
          * Indicates whether the element can be focused and where it participates in sequential keyboard navigation.
+         *
          * @protected
          */
         tabindex: {
           type: Number,
-          value: 0,
```

Comment:
**note:** It would actually be better `null` by default to avoid `undefined` at all. It would however be a breaking change so I decided not to change this behavior for now.